omnigan/trainer.py

```diff
@@ -1369,10 +1376,10 @@ def run_evaluation(self, verbose=0):
         self.train_mode()
         print("****************** Done *********************")
 
-    def save(self):
+    def save(self, save_path_name="latest_ckpt.pth"):
         save_dir = Path(self.opts.output_path) / Path("checkpoints")
         save_dir.mkdir(exist_ok=True)
-        save_path = Path("latest_ckpt.pth")
+        save_path = Path(save_path_name)
```

Comment:
We should save checkpoints twice:

1/ `epoch_n_ckpt.pth`
2/ `latest_ckpt.pth`

The 2nd will get overwritten at  every save but at least we have a super easy way of just  getting the latest, and it will always be the same whatever the exp and the number of epochs it has